src/System.Json/src/System.Json.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>$(NetFrameworkMinimum);netstandard2.0;$(NetMinimum)</TargetFrameworks>
+    <!-- System.Json is a library for compat purposes so we want to keep the same identity as it originally shipped within Silverlight. -->
+    <StrongNameKeyId>MicrosoftShared</StrongNameKeyId>
```

Comment:
Directory.Build.props applies before the project body so it gets overwritten by the one in the project.  You can always examine a build log or a preprocessed project to see how this works.  
The one in the props file would apply to other projects in that subdirectory - so that could be why it's there, or someone might have just copy/pasted and forgot to remove it.